main.tf

```diff
@@ -22,15 +31,15 @@ module "obs_staging_vpc" {
   deploy_rds_db                          = true
   xchain_settings = {
     enabled      = true
-    docker_image = var.xchain_indexer_docker_image
+    docker_image = local.xchain_indexer_staging_docker_image
     config       = "staging"
     omni_config = {
       omni_rpc = local.omni_staging_rpc
     }
   }
   blockscout_settings = {
-    blockscout_docker_image = var.staging_blockscout_docker_image
-    rpc_address             = "http://staging.omni.network:8545"
+    blockscout_docker_image = local.blockscout_staging_docker_image
+    rpc_address             = local.omni_staging_rpc
     ws_address              = "ws://staging.omni.network:8546"
```

Comment:
nit, but let's add these to locals as well, since we're now tacking rpcs changes in locals and this would need to change if the rpc changes. same with testnet ws address

Reply:
Done.